recipe/meta.yaml

```
@@ -23,28 +23,28 @@ requirements:
   build:
     - {{ compiler('cxx') }}
   host:
-    - python >=3.6
-    - setuptools
+    - python
     - pip
-    - atom >=0.6.0
-    - kiwisolver >=1.2.0
-    - ply >=3.4
-    - qtpy >=1.3
-    - cppy >=1.1.0
-    - bytecode >=0.11.0
+    - setuptools
+    - wheel
+    - setuptools_scm
+    - toml
+    - cppy 1.2.1
+    - bytecode 0.14.0
   run:
-    - python >=3.6
-    - python.app  # [osx]
-    - atom >=0.6.0
-    - pyqt
-    - qtpy >=1.3
+    - python
```

```suggestion
    - python
    - python.app # [osx]
    - pyqt
    - qtpy >=2.1.0
```
Not sure, but I think we should keep python.app.
We should also keep pyqt and qtpy because we need at least one backend.
https://github.com/nucleic/enaml/blob/0.16.0/pyproject.toml#LL41C2-L41C2